packages/app/src/api.ts

```diff
@@ -171,6 +180,126 @@ export function initializeApp(
   return newApp;
 }
 
+/**
+ * Creates and initializes a {@link @firebase/app#FirebaseServerApp} instance.
+ *
+ * The FirebaseServerApp is similar to FirebaseApp, but is intended for execution in
+ * server side rendering environments only. Initialization will fail if invoked from a
+ * browser environment.
+ *
+ * See
+ * {@link
+ *   https://firebase.google.com/docs/web/setup#add_firebase_to_your_app
+ *   | Add Firebase to your app} and
+ * {@link
+ *   https://firebase.google.com/docs/web/setup#multiple-projects
+ *   | Initialize multiple projects} for detailed documentation.
+ *
+ * @example
+ * ```javascript
+ *
+ * // Initialize a FirebaseServerApp.
```

Comment:
I wonder about using "FirebaseServerApp" as a noun on its own.

Should this be "Initialize an instance of  FirebaseServerApp" or maybe "Initialize a FirebaseServerApp object?"

Reply:
Done.